src/index.js

```diff
@@ -25,17 +25,15 @@ const Change = require('./Change.js');
 const contains = require('./contains.js');
 const indexPipelines = require('./index-pipelines.js').run;
 
-const algolia = require('./providers/algolia.js');
-const azure = require('./providers/azure.js');
-const excel = require('./providers/excel.js');
-const mapResult = require('./providers/mapResult.js');
+const excel = require('./excel.js');
+const mapResult = require('./mapResult.js');
 const recordsWrap = require('./records-wrapper.js');
 
 /**
  * List of known index providers.
  */
 const providers = [
```

Comment:
do you want to keep the multiple providers support? you could even simplify this.

Reply:
yes, just in case someone used the other 2 providers, I'd be glad if they get a warning about an unsupported provider, not a message that the excel provider can't be instantiated.